README.md

```diff
@@ -251,6 +251,8 @@ from mqt.bench import get_benchmark
 qc = get_benchmark("dj", "alg", 5)
 ```
 
+For examples can be found in the `notebooks/examples.ipynb` jupyter notebook.
```

Comment:
```suggestion
Examples can be found in the `notebooks/examples.ipynb` jupyter notebook.
```
Additionally, you can actually link to the respective file here.

Reply:
done